Source/Core/Core/IOS/ES/ES.cpp

```diff
@@ -314,6 +315,7 @@ bool ES::LaunchPPCTitle(u64 title_id, bool skip_reload)
   if (!tmd.GetContent(tmd.GetBootIndex(), &content))
     return false;
 
+  Host_PPCTitleChanged();
```

Comment:
To me, it would make more sense to put this call in `SConfig::SetRunningGameMetadata(const std::string&, u64, u16, Core::TitleDatabase::TitleType)` in ConfigManager.cpp. Any particular reason it's here instead?

Reply:
I didn't see SetRunningGameMetadata() when I implemented this. I'll move it there tonight.

Reply:
Done.